### DISCUSSION TITLE
Welcome to Grasscutter Discussions!

### DISCUSSION THREAD

**Discussion by KingRainbow44** [Melledy/Grasscutter#410] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

oh yea and support

**Comment by lunaticwhat** [comment]
seggs

**Comment by alt3ri** [comment]
s🐸

**Comment by memetrollsXD** [comment]
sexoooo

**Comment by omg-xtao** [comment]
Great.

**Comment by xmplay** [comment]
guys my dog just came up to me and said 4209???????????? how do i fix him????????

**Reply by Kazuha2022** [reply]
kill it

**Reply by LincolnFan** [reply]
check the port

**Reply by AdemJensen** [reply]
Reboot your doc gracefully:)

**Comment by alt3ri** [comment]
guys, my homie just got 4209 404 502 4301 4201 -9243
HOW TO FIX THESE SO MY HOMIE WON'T USE HIS STICK TOWARD MINE ANYMORE

**Comment by KiminaNatsuki** [comment]
sex mod release date please

**Comment by Arikatsu** [comment]
sex mod pls

**Comment by kotomei** [comment]
when can we get sex mod pls

**Comment by KormiMeiko** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> oh yea and support

I built a website to discuss grasscutters a few days ago. This is the address: http://grasscutters.club/
😊😊

**Reply by alt3ri** [reply]
> > ## 👋 Welcome!
> > We’re using Discussions as a place to connect with other members of our community. We hope that you:
> > 
> > * Ask questions you’re wondering about.
> > * Share ideas.
> > * Engage with other community members.
> > * Welcome others and are open-minded. Remember that this is a community we
> >   build together 💪.
> > 
> > To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> > oh yea and support
> 
> I built a website to discuss grasscutters a few days ago. This is the address: http://grasscutters.club/ 😊😊

<3

**Reply by Fanixtar** [reply]
not working

**Comment by Yuri-NagaSaki** [comment]
nice!

**Reply by Stevwrz** [reply]
nice!

**Comment by hit2004** [comment]
the healing of all avatars doesn`t work, pls take it into fix issues.

**Comment by selunenashor** [comment]
Can we edit gacha banner in Grasscutter server?

**Comment by 3419534380** [comment]
怎么用啊兄弟们，我是萌新完全看不懂啊啊啊啊啊

**Reply by dMxb12** [reply]
需要有一定基础才能搭建，不过已经有人做出整合包了

**Reply by MiniStar826** [reply]
整合的在哪，可以方便指个路不~

**Reply by dMxb12** [reply]
> 整合的在哪，可以方便指个路不~

baidu

**Reply by LemonXorP** [reply]
> 需要有一定基础才能搭建，不过已经有人做出整合包了

请问这东西可以用在国服吗，不知道为啥我的国际服客户端老是卡在校验那一步，一点都不带动的。

**Reply by 2045739695** [reply]
两个拼接起来的，没有整合的

**Comment by 3419534380** [comment]
Somebody help me. I don't know how to use it at all

**Comment by AliceCode987** [comment]
国服可以的，估计是最近有更新吧，不知道后续有没有拦截检测更新的功能，可以一个issues



| |
MIAIONE
Mimiks Workshop  - 
前后端构架/技术顾问/开发人员
***@***.***
|




---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2022年05月13日 12:22 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***> |
| 主题 | Re: [Grasscutters/Grasscutter] Welcome to Grasscutter Discussions! (Discussion #410) |

需要有一定基础才能搭建，不过已经有人做出整合包了

请问这东西可以用在国服吗，不知道为啥我的国际服客户端老是卡在校验那一步，一点都不带动的。

**Reply by LemonXorP** [reply]
OKK老哥就是用国服的吗？谢谢老哥，我还是老老实实去整个国服吧hhh

**Reply by LemonXorP** [reply]
老哥就是我成功配好环境上去了，然后目前看起来是没有活动、任务、以及野怪配置的，请问这个是正常的吗

**Comment by LemonXorP** [comment]
老哥有个小问题，就是我成功上去了，然后里面正常来说是不是不会有任务、活动等内容啊（然后不知道为啥合成台没了23333（虽然可以直接/give吧））




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Grasscutters/Grasscutter"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月13日(星期五) 中午1:31
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Grasscutters/Grasscutter] Welcome to Grasscutter Discussions! (Discussion #410)





 国服可以的，估计是最近有更新吧，不知道后续有没有拦截检测更新的功能，可以一个issues
 
 
 
 | |
 MIAIONE
 Mimiks Workshop  - 
 前后端构架/技术顾问/开发人员
 ***@***.***
 |
 
 
 
 
 ---- 回复的原邮件 ----
 | 发件人 | ***@***.***&gt; |
 | 日期 | 2022年05月13日 12:22 |
 | 收件人 | ***@***.***&gt; |
 | 抄送至 | ***@***.***&gt; |
 | 主题 | Re: [Grasscutters/Grasscutter] Welcome to Grasscutter Discussions! (Discussion #410) |
 
 需要有一定基础才能搭建，不过已经有人做出整合包了
 
 请问这东西可以用在国服吗，不知道为啥我的国际服客户端老是卡在校验那一步，一点都不带动的。

**Comment by lilmayofuksu** [comment]
my dog just came up to me and said DISPATCH_REGION_GET_TIMEOUT ???????????? how to fix him????????

**Comment by LemonXorP** [comment]
Sorry i don't know the answer to that question either. But it's clear that the functions of this server are not complete.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Grasscutters/Grasscutter"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年5月14日(星期六) 晚上6:22
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Grasscutters/Grasscutter] Welcome to Grasscutter Discussions! (Discussion #410)





 
my dog just came up to me and said DISPATCH_REGION_GET_TIMEOUT ???????????? how to fix him????????

**Comment by xmplay** [comment]
this is like reddit

**Reply by memetrollsXD** [reply]
Downvoted 🥴🥴

**Reply by xmplay** [reply]
> Downvoted 🥴🥴

that's a reddit moment 😔

**Comment by AZthemute** [comment]
I like cheese

**Comment by 2045739695** [comment]
我是中国人！！

**Comment by Fanixtar** [comment]
backroom hash repo leak

**Comment by LincolnFan** [comment]
目前就是这样，等待更新补全把Sent from my phone-------- Original message --------From: LemonXorP ***@***.***>Date: Sun, May 15, 2022, 5:30 PMTo: Grasscutters/Grasscutter ***@***.***>Cc: LincolnFan ***@***.***>, Comment ***@***.***>Subject: Re: [Grasscutters/Grasscutter] Welcome to Grasscutter Discussions! (Discussion #410)

老哥就是我成功配好环境上去了，然后目前看起来是没有活动、任务、以及野怪配置的，请问这个是正常的吗

**Comment by AsrieltheGoat** [comment]
SEGS!!!

**Comment by Arikatsu** [comment]
THE ONE PIECE IS REAL

**Comment by TotallyNotOndre** [comment]
Seggs plugin?
Whre do i download Seggs plugin??? i want seggs scald

**Reply by Arikatsu** [reply]
https://goku.com/

**Comment by Ferrmolina** [comment]
what?
